order.nematic API change (#1074)
glotzerlab/freud: ChangeLog.md

* The `normalize` argument to `freud.density.RDF` is now `normalization_mode`.
* The ``neighbors`` argument to ``env_neighbors`` for ``EnvironmentMotifMatch`` class.
* The ``neighbors`` argument to ``cluster_neighbors`` for ``EnvironmentCluster`` class.

> * The `freud.order.Nematic` method no longer requires `u` ("nematic director") and uses orientation vectors instead of quaternions in the `compute` method.
> 

Reviewer: ```suggestion
* `freud.order.Nematic` uses orientation vectors instead of quaternions and a nematic director.

```
